flang/lib/Lower/Bridge.cpp

```diff
@@ -4244,13 +4244,7 @@ class FirConverter : public Fortran::lower::AbstractConverter {
                       Fortran::lower::AggregateStoreMap &storeMap) {
     Fortran::lower::instantiateVariable(*this, var, localSymbols, storeMap);
     if (var.hasSymbol()) {
-      if (var.getSymbol().test(
-              Fortran::semantics::Symbol::Flag::OmpThreadprivate))
-        Fortran::lower::genThreadprivateOp(*this, var);
-
-      if (var.getSymbol().test(
-              Fortran::semantics::Symbol::Flag::OmpDeclareTarget))
-        Fortran::lower::genDeclareTargetIntGlobal(*this, var);
+      handleOpenMPSymbolProperties(*this, var);
```

Comment:
The braces are probably not required anymore.
Not sure about the name. If there is no better alternative, please document what the function does in the header.